plugin.video.itvrs/changelog.txt

```diff
@@ -0,0 +1,2 @@
+v1.0.0  (2019-6-28)
```

Comment:
changelog.txt is no longer parsed by kodi so it is an unnecessary file in the submission. Please use the `<news>` tag in the addon.xml